docs/guide/data_types/maps.rst

```diff
@@ -1,339 +1,189 @@
-    pc2_2: 1.0
-    rsun_obs: 965.3829548285768
+The `Acquiring Data <https://docs.sunpy.org/en/stable/generated/gallery/index.html#acquiring-data>`_ section
+of the Example Gallery has several examples of retrieving data using the `~sunpy.net.Fido` tool. For more details,
+check out the :doc:`/guide/acquiring_data/fido` guide.
 
-As we can see, a list of WCS and observer meta information is contained within the generated headers,
-however we may want to include other meta information including the observatory name, the wavelength and
-waveunit of the observation. Any of the keywords listed in ``header_helper.meta_keywords`` can be passed
-to the `~sunpy.map.make_fitswcs_header` and will then populate the returned MetaDict header.
-Furthermore, the following observation keywords can be passed to the `~sunpy.map.make_fitswcs_header`
-function and will be translated to the FITS standard: ``observtory``, ``instrument``,``telescope``, ``wavelength``, ``exposure``.
-
-An example of creating a header with these additional keywords::
+.. _inspecting-maps:
 
-    >>> header = sunpy.map.make_fitswcs_header(data, coord,
-    ...                                        reference_pixel = u.Quantity([5, 5]*u.pixel),
-    ...                                        scale = u.Quantity([2, 2] *u.arcsec/u.pixel),
-    ...                                        telescope = 'Test case', instrument = 'UV detector',
-    ...                                        wavelength = 1000*u.angstrom)
-    >>> header  # doctest: +SKIP
-    MetaDict([('wcsaxes', 2),
-          ('crpix1', 5.0),
-          ('crpix2', 5.0),
-          ('cdelt1', <Quantity 2. arcsec2 / pix2>),
-          ('cdelt2', <Quantity 2. arcsec2 / pix2>),
-          ('cunit1', Unit("arcsec")),
-          ('cunit2', Unit("arcsec")),
-          ('ctype1', 'HPLN-TAN'),
-          ('ctype2', 'HPLT-TAN'),
-          ('crval1', 0.0),
-          ('crval2', 0.0),
-          ...
-          ('date-obs', '2013-10-28T00:00:00.000'),
-          ('hgln_obs', 0.0),
-          ('hglt_obs', 4.7711570596394015),
-          ('dsun_obs', 148644585949.4918),
-          ('rsun_ref', 695700.0),
-          ('rsun_obs', 965.3829548285768),
-          ('instrume', 'Test case'),
-          ('wavelnth', 1000),
-          ('detector', 'UV detector'),
-          ('waveunit', 'angstrom')])
-
-From these header MetaDict's that are generated, we can now create a custom map::
-
-    >>> my_map = sunpy.map.Map(data, header) # doctest: +SKIP
-    >>> my_map.peek() # doctest: +SKIP
+2. Inspecting Maps
+==================
 
-Inspecting maps
-===============
-A map contains a number of data-associated attributes. To get a quick look at
-your map simply type::
+A Map contains a number of data-associated attributes. To get a quick look at
+your Map simply type: ::
 
     >>> my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)  # doctest: +REMOTE_DATA
     >>> my_map  # doctest: +REMOTE_DATA
-    <sunpy.map.sources.sdo.AIAMap object at ...>
```

Comment:
Was there a reason to remove the output from this?

Reply:
My thinking is that a person using the guide will have the guide open in one window and a python console in another (where they will be typing and executing each step of the guide). So, they will execute the command and read the paragraph following this code which explains what they are looking at. In the case of plots, I think it is helpful to have an example a user can reference but the summary output just seemed to take up space.

Reply:
So the issue is that these code blocks are executed as tests, so you will need to also remove the line above to stop it expecting that output to be there.

Reply:
Okay, I think I understand. Could I remove line 65 and change line 66 to: `>>> my_map  # doctest: +SKIP`?

Reply:
I am not in favour of skipping lines unless we have a good reason. I would rather remove them.

Personally, I don't think we should be removing the output from this but I will want others to chime in first.

Reply:
I think it's a different take on what a "guide" is.

Reply:
After giving this some thought, I think the output should be included. While I do approach this guide differently, it should stay consistent with the rest of the documentation.